chalk/shapes/__init__.py

```diff
@@ -37,9 +43,7 @@ def arc(radius: float, angle0: float, angle1: float) -> Diagram:
       Diagram
 
     """
-    return Primitive.from_shape(
-        Arc(radius, to_radians(angle0), to_radians(angle1))
-    )
+    return Path([ArcSegment(angle0, angle1 - angle0).scale(radius)]).stroke()
```

Comment:
I don't know how important this is, but should the `arc` function to use the same direction (either clockwise or counterclockwise) irrespective of its arguments? For example, 
```python
dia = cat(
    [
        arc(1, 0, 90).show_origin(),
        arc(1, 90, 0).show_origin(),
        arc(1, 90, 360).show_origin(),
    ],
    unit_x,
    sep=0.5
)
```
produces
![test_arc](https://user-images.githubusercontent.com/819256/183870810-15910693-1007-472a-99ff-17f10f4b01b8.png)
but maybe the second arc should be the same as the third one?

Reply:
I have no idea what is right here. As far as I can tell the different rendering libraries make different choices.